demisto_sdk/commands/common/hook_validations/script.py

```diff
@@ -72,6 +72,10 @@ def is_valid_file(self, validate_rn=True):
             ])
         return is_script_valid
 
+    def is_valid_as_deprecated(self):
+        """Check if the script is valid as a deprecated script."""
+        return self.is_valid_deprecated_script()
```

Comment:
I think you can replace `is_valid_deprecated_script` with `is_valid_as_deprecated` and remove it from the `is_valid_file` check